build.gradle

```diff
@@ -13,6 +13,8 @@ plugins {
     id 'java-library'
     id 'maven-publish'
     id 'signing'
+    id 'io.codearte.nexus-staging'  version '0.20.0'
```

Comment:
The build.gradle file is autogenerated right ?
Won't this change be overwritten in the next automatic build from the sdk-generator ?